source/help/faq/texlive.md

```diff
@@ -1,5 +1,7 @@
 # Notes about arXiv's TeXLive Version
 
+>NOTE: arXiv has switched from AutoTeX to a plain instance of TeX Live 2023. This means any package not included in the TeX Live 2023 release should have its style files included with your submission.

```

Comment:
```suggestion
>NOTE: arXiv Submission 1.5 now uses a plain instance of TeX Live 2023 instead of AutoTeX. This means any package, like hyperref, not included in the TeX Live 2023 release should have its style files included with your submission.

```